### ISSUE TITLE
Friend list

### ISSUE THREAD

**Issue by Gennoken** [fang2hou/ElvUI_WindTools#107]
```lua
13x ElvUI_WindTools\Modules\Social\FriendList.lua:245: attempt to index field '?' (a nil value)
```

**Comment by fang2hou** [comment]
Can you make a screen shot of your friend list when you meet the lua error?

**Comment by Gennoken** [comment]
It's Exile's Reach, when i'm there i get all sort of lua error, not only from WindTools but from many other addons, but once i'm out, no more errors.
closing this.

**Comment by fang2hou** [comment]
Great, and thanks for the valued information.
I will work on the bugs in Exile's Reach.

**Comment by Gennoken** [comment]
@fang2hou 
apparently it's not related to Exile's Reach, i'm still having this error.
![WoWScrnShot_101820_193730](https://user-images.githubusercontent.com/44506648/96376949-cfcd8600-1179-11eb-853f-94e6bb03595a.jpg)
You don't see it here cuz it went directly to Bugrabber

**Comment by fang2hou** [comment]
I guess the bug caused by new blizzard game, I will update the game list later.

**Comment by fang2hou** [comment]
Is that your friend was playing COD MW2?

**Comment by fang2hou** [comment]
I tried to fix it in 2.01-alpha-2, if you have the same problem after update to alpha-2, please reopen this issue.
